README.md

```diff
@@ -1,2 +1,57 @@
-# YouTube-s-video-player-page-
-First Hmtl  Project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name: YouTube-s-video-player-page
+
+> In this project we strived to create a youtube video player page clone that was responsive and aesthetically pleasing. We used HTML & CSS3(Flexbox) to achieve this.
+![screenshot](./graphics/Screenshot.png)
+
```

Comment:
- [ ] **[Optional]** Since this is your first project, this is not required, but you'll need a live preview link to display your work for the recruiters, you can use GitHub Pages or [GitHack](raw.githack.com) to do so.